packages/types/src/common/bot-profile.ts

```diff
@@ -0,0 +1,11 @@
+export interface BotProfile {
```

Comment:
I think this event is also in `message.ts`?

Reply:
The idea here is that common things reused across different events (and maybe even more generally - not just events) will exist in `src/common`. This `BotProfile` thing is exported and used in two of the `src/event` files.

Reply:
I guess I don't follow why it'd be defined in two places: isn't that superfluous?

Reply:
100% 😄 the duplicate was a mistake of mine, and it's now removed.